src/client.ts

```diff
@@ -176,20 +184,21 @@ export class Client {
     wrap = true
   ): Promise<Response | DoofinderResult> {
     const qs: string = this.buildSearchQueryString(query, params);
-    const response: Response = await this.request(this.buildUrl('/search', qs));
+    let response: Response;
```

Comment:
`search()` method only knows about the `/search` endpoint. Create a new `getItems()` method to perform requests to the `/getitems` endpoint.